index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#" class="logo"> My Logo </a>
+        <div class="menu">
+          <img src="images/svg/Menu.svg" alt="Menu" />
+        </div>
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> Portfolio </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> About </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="hover-blue-text"> Contact </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="header">
+      <div class="container">
+        <h1 class="title">
+          I'm Eduardo <br />
+          Glad to see you!
+        </h1>
```

Comment:
- [ ] This highlighted section does not use the appropriate color as shown from the figma design, please kindly make this adjustment to your project design as it is important to always match closely the figma design :+1: 
- figma design
![color](https://github.com/DHE19/portfolio-microverse/assets/89055502/35d1a999-4576-4a88-9532-cdda0d8f7e0c)